test.sh

```diff
@@ -190,6 +190,33 @@ curl --url "https://$SINGULARITY_HOSTNAME/login" \
   | tee test/login_success \
   | grep "msg = user authenticated by password"
 
+LOGIN_COOKIE="$(sed -nE 's:.*<td>Token</td>[ \t]*<td>([^<]*)</td>.*:\1:p' < test/login_success | tr -d '\n')"
```

Comment:
yeah it can, and I considered it, but I couldn't output to stdout since that is used for the html, and while I could put it in a file, I wasn't sure where to put it, but I suppose there isn't any reason it couldn't just go in the test folder.